docs/source/share_kedro_viz.md

```diff
@@ -21,116 +21,55 @@ kedro new --starter=spaceflights-pandas
 
 ```text
 fsspec[s3]>=2023.9.0
 kedro>=0.18.2
 ```
 
-Install the dependencies from the project root directory by typing the following in your terminal:
-
-```bash
-pip install -r src/requirements.txt
-```
-
-## Configure your AWS S3 bucket and set credentials
-
-You can host your Kedro-Viz project on Amazon S3. You must first create an S3 bucket and then enable static website hosting. To do so, follow the [AWS tutorial](https://docs.aws.amazon.com/AmazonS3/latest/userguide/HostingWebsiteOnS3Setup.html) to configure a static website on Amazon S3.
-
-Once the S3 bucket is created, you'll need to create an Identity and Access Management (IAM) user account, user group, and generate the corresponding access keys. To do so:
-
-Sign in to the [AWS Management Console](https://console.aws.amazon.com/s3/) and create an IAM user account.
-For more information, see the official AWS documentation about [IAM Identities](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html).
-
-![](./images/kedro_viz_share_credentials1.png)
-
-Create a user group from the IAM dashboard, ensuring the user group has full access to the AWS S3 policy.
-
-![](./images/kedro_viz_share_credentials2.png)
-
-For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
-
-Add the IAM user to the user group (this is only possible if the group has been created).
+There are two ways to publish and share Kedro-Viz:
 
-![](./images/kedro_viz_share_credentials3.png)
+1. **Platform-agnostic sharing with Kedro-Viz:** Using the kedro-viz build command, users can publish Kedro-Viz to any static website hosting platform such as GitHub pages, and share the URL generated. This is described in [Platform-agnostic sharing with Kedro-Viz](./platform_agnostic_sharing_with_kedro_viz)
 
-Select the user, then select `Create access key`. Follow the steps and create your keys.
+2. **Publish and share Kedro-Viz automatically:** You can automate the publishing and sharing of your Kedro-Viz through three different cloud providers: AWS, Azure, and GCS. Detailed instructions for each provider are outlined in the sections below.
 
-![](./images/kedro_viz_share_credentials4.png)
+* [Publish and share Kedro-Viz on AWS](./publish_and_share_kedro_viz_on_aws)
+* [Publish and share Kedro-Viz on Azure](./publish_and_share_kedro_viz_on_azure)
+* [Publish and share Kedro-Viz on GCP](./publish_and_share_kedro_viz_on_gcp)
 
 
-Once that's completed, you'll need to set your AWS credentials as environment variables in your terminal window, as shown below:
-
-```bash
-export AWS_ACCESS_KEY_ID="your_access_key_id"
-export AWS_SECRET_ACCESS_KEY="your_secret_access_key"
+```{note}
+From Kedro-Viz version 8.0.0, in addition to AWS, you will be able to publish and share your Kedro-Viz on Azure and GCP.
```

Comment:
May be we can remove this note from here and keep it in cloud provider specific pages like for Azure - From Kedro-Viz version 8.0.0, you will be able to publish and share your Kedro-Viz on Azure.

Reply:
Resolved.